packages/js/github-actions/actions/get-plugin-releases/src/get-plugin-releases.js

```diff
@@ -0,0 +1,122 @@
+/**
+ * External dependencies
+ */
+import fetch from 'node-fetch';
+import core from '@actions/core';
+
+/**
+ * Internal dependencies
+ */
+import handleActionErrors from '../../../utils/handle-action-errors';
+
+async function getPluginReleases() {
+	const slug = getInput( 'slug' );
+	const apiEndpoint = getAPIEndpoint( slug );
+
+	fetch( apiEndpoint )
```

Comment:
:construction: 
```suggestion
return fetch( apiEndpoint )
```
Otherwise, there's nothing to `await` form, and we [notify finish before any work is even started](https://github.com/woocommerce/grow/actions/runs/4753361988/jobs/8444847812#step:2:10):